docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -267,10 +268,12 @@ To use GPU acceleration on Linux, several environment variables are required or
 
       .. code-block:: bash
 
-         # configures OneAPI environment variables
+         # Required step. Configure OneAPI environment variables
          source /opt/intel/oneapi/setvars.sh
 
+         # Recommended Environment Variables
          export LD_PRELOAD=${LD_PRELOAD}:${CONDA_PREFIX}/lib/libtcmalloc.so
+         export USE_XETLA=OFF
```

Comment:
May need some verification. @rnwang04 @plusbang Do we need this variable on Max?

Reply:
I think we can remove this.

Reply:
In this case, we should update run-max-gpu.sh (which still contains "export USE_XETLA=OFF") so that scripts are consistent with documentation. Maybe in a separate PR?

Reply:
Yeah we should update run-max-gpu.sh. Both ok for update in this pr or a separate PR.